spyder/plugins/completion/kite/client.py

```diff
@@ -170,4 +172,17 @@ def perform_request(self, req_id, method, params):
                 converter = getattr(self, converter_name)
                 if response is not None:
                     response = converter(response)
-        self.sig_response_ready.emit(req_id, response or {})
+        if not isinstance(response, dict):
+            if not running_under_pytest():
```

Comment:
@andfoy running on Windows I get the following traceback:

```
Traceback (most recent call last):
  File "C:\Users\Daniel\Documents\Spyder\Spyder otros\edgar\spyder\spyder\plugins\completion\kite\client.py", line 185, in perform_request
    "center</a>").format(method, response))
TypeError: critical(QWidget, str, str, buttons: Union[QMessageBox.StandardButtons, QMessageBox.StandardButton] = QMessageBox.Ok, defaultButton: QMessageBox.StandardButton = QMessageBox.NoButton): argument 1 has unexpected type 'KiteClient'
```

Which I think is related with my previous comment regarding test failing. Also seems like some PEP8 issues remain